lib/k8s/api/metav1.rb

```diff
@@ -10,6 +10,9 @@ module API
     module MetaV1
       # @see https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#TypeMeta
       class Resource < Struct
+        # @!macro [attach] attribute
```

Comment:
This makes all of the api struct docs get documentation for their attributes.

Before:

![image](https://user-images.githubusercontent.com/224971/57526737-daee7680-7336-11e9-8f3d-cc3419248212.png)

After:

![image](https://user-images.githubusercontent.com/224971/57526756-e5a90b80-7336-11e9-85c5-35fc9f20281a.png)